Tests/src/InternalTests/WebSocketClientTests.cpp

```diff
@@ -40,7 +40,7 @@ CSP_INTERNAL_TEST(CSPEngine, WebSocketClientTests, SignalRClientStartStopTest)
 	LogIn(UserSystem, UserId);
 
 	// Start
-	auto* WebSocket = WebSocketStart("wss://ogs-odev-internal.magnoboard.com/mag-multiplayer/hubs/v1/multiplayer?id=");
+	auto* WebSocket = WebSocketStart("wss://ogs-multiplayer-odev-internal.magnoboard.com/mag-multiplayer/hubs/v1/multiplayer?id=");
```

Comment:
nitpick: I know they're just tests, but I would still argue against any hardcoded URLs in code.

Reply:
This shouldn't work. The URL is pointing to the old Dev environment which doesn't exist anymore. Can we take this opportunity to change the URL to `wss://ogs-multiplayer-internal.magnopus-dev.cloud/mag-multiplayer/hubs/v1/multiplayer`?

Reply:
Sure thing. Does "-multiplayer" always come before other strings like "-internal"?

Reply:
awesome, thank you for confirming